### ISSUE TITLE
Run a join query and expose it as  view to be published via GeoServer

### ISSUE THREAD

**Issue by bermud** [GES771-SP20/Final_Project#6]
geometry
name of state
property and values

**Comment by scottclement14303** [comment]
Prof. Bermudez,

I am having trouble running the sql statement that will be the basis for the view we want to create... I think the problem I am having is with a database setting that limits either the time spent processing a query or the number of rows returned, and not with the sql statement itself. 

Here is a pic of the query and return for three days' worth of data:
![image](https://user-images.githubusercontent.com/61480706/79523055-536fb400-802b-11ea-921a-4d84e9ac368a.png)

Here is a pic of the query and return for four days' worth of data:
![image](https://user-images.githubusercontent.com/61480706/79523190-93cf3200-802b-11ea-871e-aa86f450d754.png)

To summarize: I get a return for three or fewer days of data, but get nothing for four days or more. I also get no rows returned if I eliminate the WHERE statement altogether, which is actually what I want to do since we'd like to create one view that joins all 2200+ rows in our covid table with the geometries in the states table.

Any ideas on what may be causing this problem?
